apps/meteor/app/ui-utils/client/lib/messageActionDefault.ts

```diff
@@ -241,9 +241,9 @@ Meteor.startup(async () => {
 				},
 			});
 		},
-		condition({ subscription, room }) {
+		condition({ subscription, room, message }) {
```

Comment:
I think you can get the user from this object, instead of using `Meteor.userId()`

Reply:
Done!